pkg/jobs/pool.go

```diff
@@ -320,21 +378,23 @@ func (p *Pool) watchJobs() {
 		if p.GetStatus() == PoolStatusRunning.String() {
 			// The pool is running, wait for any signal that a job went down
 			<-p.stopJobsChan
-			p.logger.
-				WithField("jobName", p.cronJobs[p.failedJob].GetName()).
-				WithField("failed job", p.failedJob).
-				Debug("Job failed, stop all jobs")
+			if job, found := p.getCronJob(p.getFailedJob()); found {
+				p.logger.
+					WithField("jobName", job.GetName()).
+					WithField("failed job", p.getFailedJob()).
```

Comment:
We should make this naming convention standard. The jobName field should also be job-name, right?

Reply:
this seems a little out of scope for this review. If I search for "withfield" all over, there are many things "misnamed", e.g. endTimestamp, eventType, totalBytes, etc. If these are all non-conventional, I suggest a separate ticket be created and they all be changed at once, because I assume this is a separate testing effort. Agree or disagree?

Reply:
Agreed, I was just looking for consensus on working on a standard way of making these. Lower camel case or dash separated? This change should abide by that, but we need to update others separately.

Reply:
Created story for this https://jira.axway.com/browse/APIGOV-22962 .